administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -43,9 +43,9 @@
 		border: 0 !important;
 	}
```

Comment:
change this class to 
```css
#reference-association, #target-association {
 		width: 100%;
 		height: 1500px;
  		border: 0 !important;
  	}
```

why? because, as it is, you are also saying the modal list iframe as this height